src/ophyd_async/core/device_save_loader.py

```diff
@@ -177,8 +177,22 @@ def set_signal_values(
     for phase_number, phase in enumerate(values):
         # Key is signal name
         for key, value in phase.items():
+            # Skip ignored values
+            if value is None:
+                continue
+
             if key in signals:
                 yield from abs_set(
                     signals[key], value, group=f"load-phase{phase_number}"
                 )
+
         yield from wait(f"load-phase{phase_number}")
+
+
+def load_device(device: Device, path: str):
+    """
+    Sets all PV's to a device using a previously saved configuration
+    """
+    values = load_from_yaml(path)
+    signals_to_set = walk_rw_signals(device)
+    yield from set_signal_values(signals_to_set, values)
```

Comment:
If we're going to do this, we should add a `save_device` function that does the opposite. I've noticed for some reason this function currently lives in the tests for device_save_loader...

Reply:
The initial idea was to have generic functions which you could construct save/load plans from, functions like `walk_rw_signals` and `load_from_yaml` and `set_signal_values`, then save/load plans could be constructed from these for a specific use-case. We later realised the the `load_device` plan would be generic for all devices, so we might as well include it here. 
The `save_device` plan is not as generic as it will depend on the device-specific phases, which is why I didn't include it in this PR. I guess if most devices don't have a phase order then it makes sense to include `save_device` here too

Reply:
I guess we could have generic load and save plans, but pass an optional `sorter` function to it that takes the saved values and sorts them into phases?